clrs/_src/baselines_test.py

```diff
@@ -86,6 +87,7 @@ def _remove_permutation_from_spec(spec):
 
 class BaselinesTest(parameterized.TestCase):
 
+  @unittest.skip("We don't support decode_hints=encode_hints=True nor MPNN for now")
```

Comment:
I would really like to remove the skips here and for `test_multi_algorithm_idx`. We should probably implement our approach for all the processors (I assume changes should be minimal?). Even if we don't need to support, `decode_hints=encode_hints=True` we shouldn't break backward compatibility/the ability to run the default CLRS (e.g. as a baseline).

Note that those parameters may not be why these tests fail (see https://github.com/jonasjuerss/INAR/pull/4#issuecomment-2538277129) , they're just obvious reasons why I wouldn't expect these tests to pass.